### ISSUE TITLE
有人通过run.py成功训练么?

### ISSUE THREAD

**Issue by wuzhh8** [luodhhh/ModernTCN#10]
moderntcn.py 中，218行的代码，缩进是不是应该for循环内？这么改的话 运行时会报index out of range的错误。改过之后 还是会报错出现mat1和mat2不匹配的问题。
214 if self.num_stage > 1:
            for i in range(self.num_stage-1):
                downsample_layer = nn.Sequential(
                    nn.BatchNorm1d(dims[i]),
                    nn.Conv1d(dims[i], dims[i + 1], kernel_size=downsample_ratio, stride=downsample_ratio),
                )
218       self.downsample_layers.append(downsample_layer)

**Comment by wodeqiansuihan** [comment]
目前我还没有，我报别的错误

**Comment by YangJingyuan99** [comment]
如果你运行的是short_term那个代码，可以尝试把use_multi_scale设置为false，我和你是一样的问题，把这个参数改了之后就好了。只能说这个代码的鲁棒性很差😂
